pkgs/applications/misc/blender/default.nix

```diff
@@ -72,7 +72,7 @@
   pkg-config,
   potrace,
   pugixml,
-  python3Packages, # must use instead of python3.pkgs, see https://github.com/NixOS/nixpkgs/issues/211340
```

Comment:
This needs to be reverted

Reply:
I removed the change that makes Blender use Python 3.11. You can discuss that with @amarshall in https://github.com/NixOS/nixpkgs/pull/326044.